json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -84,7 +88,15 @@ protected function autoupdate_translations_off() {
 	}
 
 	protected function activate() {
+		$permission_error = false;
 		foreach ( $this->plugins as $plugin ) {
+			if ( ! $this->current_user_can( 'activate_plugin', $plugin ) ) {
```

Comment:
I think we should move this checks to  `validate_call`, if we do it here we could throw an error after turning on/off autoupdate successfully

Reply:
We are still doing those checks. This is checking on a per plugin basis if the user is able to activate/deactive a plugin. it would be really hard to do it on a global scale.